ecommerce/extensions/analytics/utils.py

```diff
@@ -204,3 +205,29 @@ def get_google_analytics_client_id(request):
         return '.'.join(google_analytics_cookie.split('.')[2:])
 
     return None
+
+
+def get_utm_session_parameters():
+    """
+    Return utm parameters from the cookie or current url request.
+    The url values have priority.
```

Comment:
This is not applying. @andrey-canon

Reply:
??

Reply:
You say the URL params have priority but the code is doing nothing to do what it is saying here. @andrey-canon

Reply:
line 227

Reply:
By overwritten, but I think is better to explain explicitly, URL parameters will overwrite Cookie parameters. @andrey-canon